### DISCUSSION TITLE
GoPro Hero 8 suddenly stopped working

### DISCUSSION THREAD

**Discussion by Sixilef** [gopro/labs#1214] [Q&A] [answered]
Hello, currently im facing a massive problem on my GoPro Hero 8, it was working normally suddenly it stopped without any cause
gopro status now is a red light only on the back frozen (not blinking) and both screens are off and the camera is not responding to any button click
i've tryed multiple solutions; reseting it by clicking the button for 10/20 seconds, putting update folder on the sd card, removing the battery and using cable, removing sd card, force reset by creating a txt folder Factory_Reset, changing battery but all in the vain
what im sure about it the gopro is working and its not completly dead because while trying all these solutions there was a moment the back mini screen showed recording status for about 10seconds and it turned off again
i've contacted the gopro support and im still waiting for their answer
while that, if you guys have any solution or anyone faced that kind of problem before, please let me know what to do
and thanks!
![WhatsApp Image 2024-12-29 at 14 43 53_279716b7](https://github.com/user-attachments/assets/30ad46a2-8ede-4292-9cae-122eabce39ff)
![WhatsApp Image 2024-12-29 at 14 43 53_d3fb23f9](https://github.com/user-attachments/assets/6cc8fdde-aa0f-4c6b-b442-0940e2ce8f19)

**Comment by dnewman-gpsw** [comment, marked as answer]
I don't have any solution. Just in case, you mean factory_reset.txt file not a folder in the root of the SD card.

As this is unrelated to Labs firmware, I'm closing this thread. This is not a general GoPro support site, no GoPro support staff are here.

**Reply by Sixilef** [reply]
i am sorry about it, i meant factory_reset.txt but nothing changes
writing this here, i was thinking it was a code file to put in the camera or something like that
thank you for your answer!
